lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -241,6 +241,19 @@ sub machine {
     return $self->settings_hash->{MACHINE};
 }
 
+sub obs_url() {
+    return "https://build.opensuse.org/";
+}
+sub obs_prj() {
+    my ($self) = @_;
+    return undef unless ($self->name =~ /Staging:[A-Z]/);
+    return "openSUSE:Factory:$&";
+}
+sub obs_prj_url() {
+    my ($self) = @_;
+    return $self->obs_url . "project/show/" . $self->obs_prj;
+}
+
```

Comment:
I really don't like prototypes, besides I don't think prototypes are even considered for method calls:
>Method calls are not influenced by prototypes either, because the function to be called is indeterminate at compile time, since the exact code called depends on inheritance.

and `return undef` since it is 1) unnecessary, 2) it changes subroutine true/false return value depending if called in scalar or array context. Thus I consider it a bad habit.